pkg/tfgen/generate.go

```diff
@@ -879,14 +883,19 @@ func (g *Generator) gatherResources() (moduleMap, error) {
 	}
 	modules := make(moduleMap)
 
+	_, failBuildOnProviderMapError := os.LookupEnv("PULUMI_PROVIDER_MAP_ERROR")
```

Comment:
Can we call this once and add it as a flag to `Generator`? Or even just in `gatherPackage()` and pass it as an arg to these two functions?

Reply:
This is purely a test - I think we will actually generate the resource if it doesn't exist going forward but I am trying to get an idea (when I build locally) on how many (if any) are missing